docs/api-and-integration/metadata/jira.md

```diff
@@ -1,55 +1,89 @@
+* Learn about [Connecting Jira Cloud and Octopus Deploy](#connecting-jira-cloud-and-octopus-deploy)
 
-1. Install the Octopus Deploy plugin in Jira.
+---
+
+## Connecting Jira Server and Octopus Deploy
+
+This section describes how to configure Octopus Deploy to connect to Jira Server. In all cases for this integration, connections are always made to Jira (either from Octopus server itself or via links in the browser). There are no cases where Jira connects to Octopus.
+
+This means you can configure any Octopus instance, whether self-hosted or cloud, to use the Jira integration. The only network connectivity requirements are that your Octopus server and your browser can connect to the Jira Base URL discussed below.
+
+If you are using Jira Server, see [Connecting Jira Cloud and Octopus Deploy](#connecting-jira-cloud-and-octopus-deploy).
+
+<!-- I've left this blank so we can review/agree upon the Jira Cloud copy first and then strip it appropriately to suit the Jira Server copy. -->
+
+
+## Connecting Jira Cloud and Octopus Deploy
+
+This section describes how to configure Octopus Deploy to connect to Jira Cloud. In all cases for this integration, connections are always made to Jira (either from Octopus server itself or via links in the browser). There are no cases where Jira connects to Octopus.
+
+This means you can configure any Octopus instance, whether self-hosted or cloud, to use the Jira integration. The only network connectivity requirements are that your Octopus server and your browser can connect to the Jira Base URL discussed below.
+
+Further, there is additional functionality available for Jira Cloud customers via the Octopus Deploy plugin for Jira that's available from the [Atlassian Marketplace](https://marketplace.atlassian.com/apps/1220376/octopus-deploy-for-jira) and the following steps describe how to configure this. This functionality enables teams to view release and deployment details from Octopus directly in Jira issues. If you are using Jira Server, see [Connecting Jira Server and Octopus Deploy](#connecting-jira-server-and-octopus-deploy).
 
-    Add the `Octopus Deploy for Jira` app from the [Atlassian Marketplace](https://marketplace.atlassian.com/apps/1220376/octopus-deploy-for-jira) and then click the 'Get Started' button to configure it.
+![Jira Issue with deployments](jira-issue-with-deployments.png "width=500")
 
-    Note: Keep this configuration page open while you navigate to Octopus in the next step as you need to copy values from one page to the other.
+1. Install the Octopus Deploy plugin in your Jira Cloud instance.
+
+    Add the `Octopus Deploy for Jira` app directly from the [Atlassian Marketplace](https://marketplace.atlassian.com/apps/1220376/octopus-deploy-for-jira) and then click the 'Get Started' button to configure it. Alternately, this is available in Jira under **{{Jira Settings,Find new apps}}**.
```

Comment:
```suggestion
    From the Atlassian Marketplace, add the [Octopus Deploy for Jira](https://marketplace.atlassian.com/apps/1220376/octopus-deploy-for-jira) app and click 'Get Started' to configure it. Alternately, the app is also available in Jira by navigation to **{{Jira Settings,Find new apps}}**.
```
Minor tweaks to put the first thing they need (the marketplace) first, then the second thing (the app) second.